pkg/api/timestamp.go

```diff
@@ -62,15 +82,35 @@ func TimestampResponseHandler(params timestamp.GetTimestampResponseParams) middl
 	}
 
 	// Create response
-	ctx := params.HTTPRequest.Context()
+	httpReq := params.HTTPRequest
+	ctx := httpReq.Context()
 	resp, err := RequestFromRekor(ctx, *req)
 	if err != nil {
 		return handleRekorAPIError(params, http.StatusInternalServerError, err, failedToGenerateTimestampResponse)
 	}
 
-	// TODO: Upload to transparency log and add entry UUID to location header.
-	log.Logger.Errorf("generated OK")
-	return timestamp.NewGetTimestampResponseOK().WithPayload(ioutil.NopCloser(bytes.NewReader(resp)))
+	// Upload to transparency log and add entry UUID to location header.
+	paramReq := *httpReq
+	paramReq.URL.Path = "/api/v1/log/entries"
```

Comment:
I might use the URL builder from the code generation here VS hard coding this. I think it would look like:

```go
cleURL := entries.CreateLogEntryURL{}
paramReq.URL = cleURL.Must(cleURL.Build())
```